src/scope.rs

```diff
@@ -640,10 +640,25 @@ macro_rules! impl_scope_drop {
 }
 
 impl_scope_drop!(<'s, 'p, P> ContextScope<'s, P>);
-impl_scope_drop!(<'s, C> HandleScope<'s, C> );
-impl_scope_drop!(<'s, 'e, C> EscapableHandleScope<'s, 'e, C> );
 impl_scope_drop!(<'s, P> TryCatch<'s, P> );
```

Comment:
I didn't implement this for `TryCatch` because it's a wrapper around another `HandleScope`, ergo, the isolate must have been entered already. Does that sound, uh, sound?